### PR TITLE
MODNOTIFY-75 update RMB and vertx versions

### PR DESCRIPTION
**Pull request by roman-barannyk**
## Approach

- Update to RMB v30.0.1
- Update vertx to 3.9.1
- Do all related changes according to RMB upgrading notes

Resolves: [MODNOTIFY-75](https://issues.folio.org/browse/MODNOTIFY-75)

### CODE REVIEW

File: src/main/java/org/folio/rest/impl/NotificationsResourceImpl.java [hunk 1]
```diff
@@ -281,6 +282,13 @@ public void postNotify(String lang,
     if (id == null || id.trim().isEmpty()) {
       id = UUID.randomUUID().toString();
     }
+    if (!UuidUtil.isUuid(id)) {
+      Errors valErr = ValidationHelper.createValidationErrorMessage(
+        "id", id, "invalid input syntax for type uuid");
+      asyncResultHandler.handle(succeededFuture(PostNotifyResponse
+        .respond422WithApplicationJson(valErr)));
+      return;
+    }
```
Comment (OleksandrVidinieiev) [inline src/main/java/org/folio/rest/impl/NotificationsResourceImpl.java:291]: Why is this a part of RMB update?

Comment (roman-barannyk) [reply src/main/java/org/folio/rest/impl/NotificationsResourceImpl.java:291]: Id validation was removed from PostgresClient. In current implementation most of tests were failed, it doesn't return validation error. Current version of PostgresClient uses UUID.fromString() method under the hood. It doesn't check UUID correctly.

File: src/main/java/org/folio/rest/impl/NotificationsResourceImpl.java [hunk 2]
```diff
@@ -465,20 +480,26 @@ public void deleteNotifyById(String id, String lang, Map<String, String> okapiHe
...
       return;
     }
-
+    if (!UuidUtil.isUuid(id)) {
+      Errors valErr = ValidationHelper.createValidationErrorMessage(
+        "id", id, "invalid input syntax for type uuid");
+      asyncResultHandler.handle(succeededFuture(PostNotifyResponse
+        .respond422WithApplicationJson(valErr)));
+      return;
+    }
```
Comment (alexanderkurash) [inline src/main/java/org/folio/rest/impl/NotificationsResourceImpl.java:489]: Extract method?

Comment (roman-barannyk) [reply src/main/java/org/folio/rest/impl/NotificationsResourceImpl.java]: done

**Pull request merged**